addons/l10n_tr_nilvera_einvoice/static/src/components/actionable_errors/actionable_errors.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="l10n_tr_nilvera_einvoice.ActionableErrors" t-inherit="account.ActionableErrors" t-inherit-mode="extension">
+        <xpath expr="//div[@role='alert']" position="attributes">
+            <attribute name="t-att-class">Object.values(this.props.record.data[this.props.name]).some(e => e.danger) ? 'alert-danger' : 'alert-warning'</attribute>
```

Comment:
this.props.record.data[this.props.name] is it the same as using the getter errorData

also will it not change the whole ActionableErrors for every company when the module is installed ?